vaadin-chip-combobox/src/main/java/software/xdev/vaadin/chips/ChipComboBox.java

```diff
@@ -224,6 +226,11 @@ public String getLabel()
 		return this;
 	}
 	
+	public ChipComboBox<T> setLabel(final String label)
```

Comment:
Please do the following:
* The setter should return ``void``, so that it represents a "Java Standard Setter"
* Move all the logic from ``with..`` to ``set...``, so that the ``with...`` method looks like this:
```java
	public ChipComboBox<T> withLabel(final String label)
	{
		this.setLabel(label)
		return this;
	}
```